.github/workflows/gh-pages.yml

```
@@ -0,0 +1,36 @@
+name: Publish Storybook
+on:
+  push:
+    branches:
+      - develop
+      - EGRC-275
+
+jobs:
+  publish:
+    name: Publish Storybook to GitHub Pages
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Setup NodeJS
+        uses: actions/setup-node@v2
+        with:
+          node-version: 14
+          cache: "npm"
+      - name: Globally update npm
+        run: npm install -g npm@latest
+      - name: Install dependencies
+        run:
+          npm install
+      - name: Build Storybook
+        run:
+          npm run build-storybook
+      - name: Deploy to Pages
+        uses: JamesIves/github-pages-deploy-action@4.1.4 
+        with:
+          branch: storybook-docs
+          folder: storybook-static
+          clean: true
```

```suggestion
          clean: true
          single-commit: true
```

I would kind of like to see us keep a flat history on this branch (since it's not our `main`/`develop` branch) and is automatically generated. The actual history is in the source files. If we were doing something like pushing to the `docs/` folder on `main` then it'd make sense to keep all history; but this is basically just plain file storage.